Cargo.toml

```diff
@@ -14,4 +14,8 @@ regex = "0.2"
 
 [[bin]]
 name = "find"
-path = "src/find/main.rs"
\ No newline at end of file
+path = "src/find/main.rs"
+
+[[bin]]
+name = "testing-commandline"
+path = "src/testing/commandline/main.rs"
```

Comment:
Very minor nit: file is missing newline at end.

I'm wondering whether the testing-commandline should be made conditional, as it doesn't ship in production. The easiest way to do that would be to use a feature.